ios/brave-ios/Sources/Brave/Frontend/Browser/Tab.swift

```diff
@@ -67,6 +67,17 @@ protocol URLChangeDelegate {
   func tab(_ tab: Tab, urlDidChangeTo url: URL)
 }
 
+struct RewardsTabChangeReportingState {
+  /// Set to true when the resulting page was restored from session state.
+  var isSessionStateRestored = false
+  /// Set to true when the resulting page navigation is not a reload or a
+  /// back/forward type.
+  var isNewNavigation = true
+  /// Set to true when the resulting page is showing an error interstitial or
+  /// the resulting response from the web page had a 4xx or 5xx status code
```

Comment:
nit: for consistency it would be great to add a full stop for this comment.

Reply:
Fixed